worldedit-core/src/main/java/com/sk89q/worldedit/command/BrushCommands.java

```diff
@@ -272,6 +273,36 @@ public void smoothBrush(Player player, LocalSession session,
         ToolCommands.sendUnbindInstruction(player, UNBIND_COMMAND_COMPONENT);
     }
 
+    @Command(
+        name = "snowsmooth",
+        desc = "Choose the snow terrain softener brush",
+        descFooter = "Example: '/brush snowsmooth 5 1 3'"
+    )
+    @CommandPermissions("worldedit.brush.snowsmooth")
+    public void snowSmoothBrush(Player player, LocalSession session,
+                                @Arg(desc = "The radius to sample for softening", def = "2")
+                                    double radius,
+                                @Arg(desc = "The number of iterations to perform", def = "4")
+                                    int iterations,
+                                @Arg(desc = "The number of snow blocks under snow", def = "1")
+                                    int snowBlockCount,
+                                @Arg(desc = "The mask of blocks to use for the heightmap", def = "")
```

Comment:
I think `mask` belong as an `@ArgFlag(name = 'm')`, like with other commands. It's different from `//smooth`, but it's more usable IMO. If you change it, also change `//snowsmooth`.